spec-main/index.js

```diff
@@ -75,16 +75,18 @@ app.whenReady().then(() => {
 
   walker.on('end', () => {
     const runner = mocha.run(() => {
-      if (isCi && runner.hasOnly) {
-        try {
-          throw new Error('A spec contains a call to it.only or describe.only and should be reverted.')
-        } catch (error) {
-          console.error(error.stack || error)
+      process.nextTick(() => {
```

Comment:
drop a comment here about why we're calling this in a later tick?


also it's a bummer that these two runners are copy/pasted from each other